src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationsHearingResultAwardResourceIntTest.kt

```diff
@@ -654,15 +676,15 @@ class AdjudicationsHearingResultAwardResourceIntTest : IntegrationTestBase() {
           .exchange()
           .expectStatus().isOk
           .expectBody()
-          .jsonPath("awardResponsesCreated[0].sanctionSequence").isEqualTo(4)
+          .jsonPath("awardResponsesCreated[0].sanctionSequence").isEqualTo(8)
```

Comment:
shows sanction sequence allows gaps (4 is not used but will choose 8 which is 1 plus highest)